Cargo.toml

```diff
@@ -65,14 +65,14 @@ windows-targets = "0.52"
 # wasm_js
 [target.'cfg(all(getrandom_backend = "wasm_js", target_arch = "wasm32", any(target_os = "unknown", target_os = "none")))'.dependencies]
 wasm-bindgen = { version = "0.2.96", default-features = false }
-js-sys = "0.3.73"
+js-sys = { version = "0.3.73", default-features = false }
 [target.'cfg(all(getrandom_backend = "wasm_js", target_arch = "wasm32", any(target_os = "unknown", target_os = "none")))'.dev-dependencies]
 wasm-bindgen-test = "0.3"
 
 [features]
 # Implement std::error::Error for getrandom::Error and
 # use std to retrieve OS error descriptions
-std = []
+std = ["wasm-bindgen/std", "js-sys/std"]
```

Comment:
We do not use any items dependent on `std` from these crates, no? And users should not depend on `getrandom` to enable the `std` feature for these crates.